_src/_posts/2018-01-04-importance-of-serving-ads.markdown

```diff
@@ -12,7 +12,7 @@ author: jdemonasterio
 
 <!--excerpt.start-->
 
-[Jampp][jampp][^1] helps customers boost their mobile sales with performance marketing. One of the most challenging problems of the industry is the impact quantification an ad has on users. A poorly tackled problem is understanding when a user has been affected enough by ads, and why acquiring clicks is not always best aligned with advertiser's interests.
+[Jampp][jampp][^1] helps customers boost their mobile sales with performance marketing. One of the most challenging problems of the industry is the impact quantification an ad has on users. A poorly tackled problem is understanding when a user has been affected enough by ads, and why acquiring clicks is not always best aligned with performance advertiser's interests.
```

Comment:
is quantifying the impact that an ad has on users

best aligned with performance advertiser's interests. >> this sentence does not make sense.

below, in general user should be gender agnostic. Use "they" instead of "he"